src/Networking/v2/Models/LoadbalancerStat.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace OpenStack\Networking\v2\Models;
+
+use OpenStack\Common\Resource\Retrievable;
+use OpenStack\Common\Resource\OperatorResource;
+
+/**
+ * Represents Neutron v2 Loadbalancer Stats
+ *
+ * @property Api $api
+ */
+class LoadbalancerStat extends OperatorResource implements Retrievable
+{
+    /**
+     * @var integer
+     */
+    public $bytesIn;
+
+    /**
+     * @var integer
```

Comment:
We personally implemented our own BigInt class to handle things like this. It was a pain in the butt, but it was a valid workaround at the time. Perhaps we could do something here as well? (or just use one of the `off-the-github` implementations.

I'm not a fan of using a `string` type here, personally.